independent-publisher-connectors/Qdrant/apiProperties.json

```diff
@@ -0,0 +1,45 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "host": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Host",
+          "description": "The Hostname of the Qdrant instance.",
+          "tooltip": "Provider your hostname",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#090E1A",
```

Comment:
Please change the iconBrandColor as per the template in the below link. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json#L4